packages/bot/src/services/tradeEngine/trade/Balance.js

```diff
@@ -12,23 +12,25 @@ export default Engine =>
                     balance: { balance: b, currency },
                 } = r;
 
-                balanceStr = getFormattedText(b, currency);
+                balance_string = getFormattedText(b, currency);
 
-                info({ accountID: this.accountInfo.loginid, balance: balanceStr });
+                info({ accountID: this.accountInfo.loginid, balance: balance_string });
             });
         }
 
         // eslint-disable-next-line class-methods-use-this
         getBalance(type) {
-            const { scope } = this.store.getState();
-            const { balance } = this;
+            const { client } = ScratchStore.instance.root_store.core;
+            const { scope }  = this.store.getState();
+            const balance    = this.balance || client.balance || 0;
+            let value        = balance;
```

Comment:
@aaron-binary I think you can use `balance`, i don't think you need a `value`.

Reply:
Makes sense. 👍